synapse/module_api/__init__.py

```diff
@@ -903,7 +903,7 @@ def read_templates(
             A list containing the loaded templates, with the orders matching the one of
             the filenames parameter.
         """
-        return self._hs.config.read_templates(
+        return self._hs.config.server.read_templates(
```

Comment:
This was a real since #10985 was merged for v1.45.0rc1 -- `read_templates` would not have worked for modules since then.

Reply:
at least you're doing some good typing work with non-academic benefits! :)